### PR TITLE
Frontend paddle tensor tensor multiply

### LINKED ISSUE #16623: multiply

### PR DESCRIPTION
**Pull request by nikita-jare**
Closes #16623 

Added multiply function in tensor class of Paddle frontend

### CODE REVIEW

**Comment by rishabgit** [comment]
@nikita-jare Could you please look into the requested change and the merge conflicts? Apart from these, this looks good to me to merge :)

**Comment by nikita-jare** [comment]
@rishabgit What are the requested changes? I am sorry if those are already mentioned somewhere but I cannot find it.

**Review by rishabgit — CHANGES_REQUESTED** [review]

**Comment by rishabgit** [comment]
My bad, I completely missed it was in the draft section 😅 Should be visible now

File: ivy/functional/frontends/paddle/tensor/tensor.py
[the remark this answers is not in the document]
Comment (nikita-jare) [reply ivy/functional/frontends/paddle/tensor/tensor.py:1]: Changed the name parameter value to None.

**Comment by rishabgit** [comment]
Hi @nikita-jare, thanks a lot for working on this and resolving the merge conflicts. Tests pass for me on Codespaces too. Merging this now :)

**Comment by MuhammadNizamani** [comment]
```
@with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
    def multiply(self, y, name=None):
        return paddle_frontend.multiply(self, y)
```
The implementation appears to be incorrect based on my understanding. According to my understanding, paddle.tensor.Tensor should receive functions from the backend, rather than the frontend. However, the above function is using paddle.tensor.tensor.math.py to access the backend.
and varible Y should be converted to ivy_array

**Comment by rishabgit** [comment]
> ```
> @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
>     def multiply(self, y, name=None):
>         return paddle_frontend.multiply(self, y)
> ```
> 
> The implementation appears to be incorrect based on my understanding. According to my understanding, paddle.tensor.Tensor should receive functions from the backend, rather than the frontend. However, the above function is using paddle.tensor.tensor.math.py to access the backend. and varible Y should be converted to ivy_array

Hi, following this [conversation](https://discord.com/channels/799879767196958751/998782045494976522/1118814704928821271) would explain things much better than I could, hope this helps! :)

**Comment by MuhammadNizamani** [comment]
@rishabgit thanks I got it 😄 sorry for your inconvenience

**Pull request merged**